python/cuml/tests/test_random_forest.py

```diff
@@ -518,7 +521,10 @@ def test_rf_classification_fit_and_predict_dtypes_differ(
         sk_model.fit(X_train, y_train)
         sk_preds = sk_model.predict(X_test)
         sk_acc = accuracy_score(y_test, sk_preds)
-        assert acc >= (sk_acc - 0.07)
+        # Increased tolerance to 0.10 to account for RNG variance from bias fix
+        # Variance analysis (1000 runs): mean=0.9140, range=[0.8429,0.9714]
```

Comment:
Awesome to see this. It'd be great if we standardized this practice and added this info in all or a lot more tests in general.

Reply:
I've updated the comments on measured tolerances in 5cf3fc9bb125dc58e0c1b98a880d93b8ef0ed527 .